dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/persistence/file/batch/BatchFileOrchestratorTest.kt

```diff
@@ -340,14 +340,14 @@ internal class BatchFileOrchestratorTest {
         assumeTrue(fakeRootDir.listFiles().isNullOrEmpty())
         var previousFile = testedOrchestrator.getWritableFile()
 
-        for (round in 1 until 5) {
+        repeat(5) {
```

Comment:
not sure if is affects the test, but it should be `repeat(4)` here, because `until` above is exclusive for the right boundary.